experiment.ipynb

```diff
@@ -36,10 +36,7 @@
           "user_tz": 240
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        weight=Variable(torch.FloatTensor([1/sum(outcomes['In-hospital_death'] == 0), 
</pre><p>good catch!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timatim/GRU_D/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>+1</p>

Reply:
<p>+1 agaain</p>